src/diracx/db/__init__.py

```diff
@@ -1,7 +1,7 @@
 __all__ = ("AuthDB", "JobDB", "JobLoggingDB", "SandboxMetadataDB")
 
-from .auth.db import AuthDB
-from .jobs.db import JobDB, JobLoggingDB
-from .sandbox_metadata.db import SandboxMetadataDB
+from .sql.auth.db import AuthDB
```

Comment:
Should that stay here, or do we want to expose to the routers the `sql` or `os` ? 
Since we started making the distinction, it's probably worth pursuing so a router would do `from diracx.db.sql import AuthDB`

Reply:
Yeah I agree, especially as the behavour and API is different in each case.